packages/nocodb/src/models/Integration.ts

```
@@ -146,12 +146,14 @@ export default class Integration implements IntegrationType {
       insertObj,
     );
 
-    return await this.get(
+    const int = await this.get(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)